packages/tools/examples/surfaceRenderingStack/index.ts

```diff
@@ -46,6 +41,7 @@ const volumeLoaderScheme = 'cornerstoneStreamingImageVolume'; // Loader id which
 const volumeId = `${volumeLoaderScheme}:${volumeName}`; // VolumeId with loader id + volume id
 const toolGroupIdStack = 'MY_TOOLGROUP_ID_STACK';
 const toolGroupIdVolume = 'MY_TOOLGROUP_ID_VOLUME';
+const segmentationId = 'MY_SEGMENTATION_ID';
 
 // ======== Set up page ======== //
 setTitleAndDescription(
```

Comment:
Could you initially navigate the stack to something that shows the data?  It wasn't obvious to me what to do to get it to show up.  Once I started scrolling, then I started seeing the data, so I think it is either a missing render on complete or the wrong position initially.

Reply:
I will re-work the stack segmentation with surface intersections pretty soon, but rather have the PR reviewed